ingest-app/src/cmr/ingest/api/services.clj

```diff
@@ -41,16 +42,18 @@
   (verify-service-modification-permission context :update)
   (common-enabled/validate-write-enabled context "ingest")
   (validate-service-content-type headers)
-  (api-core/generate-ingest-response
-   headers
-   (ingest/create-service context body)))
+  (let [metadata (string/trim (slurp body))]
```

Comment:
It seems we're using this pattern in a couple of places ... might be worth defining a tiny function someplace, e.g., `read-body` or `read-body!` that does the slurp and trim.